src/Paulus/Exception/Http/Standard/HttpExceptionFactory.php

```diff
@@ -0,0 +1,61 @@
+<?php
+/**
+ * Paulus - A PHP micro-framework for creating RESTful services
+ *
+ * @author      Trevor Suarez (Rican7)
+ * @author      Mark Costello (mcos)
+ * @copyright   2015 Trevor Suarez
+ * @link        https://github.com/Rican7/Paulus
+ * @license     https://github.com/Rican7/Paulus/blob/master/LICENSE
+ * @version     2.0.0
+ */
+
+namespace Paulus\Exception\Http\Standard;
+
+use Exception;
+
+/**
+ * HttpExceptionFactory
+ *
+ * Provides the means to create Api Exceptions using minimal other information.
+ *
+ * @uses Exception
+ * @package Paulus\Exception\Http\Standard
+ */
+class HttpExceptionFactory
```

Comment:
I'm almost inclined to suggest calling this `StandardHttpExceptionFactory`, due to the resulting classes and namespace (its not creating Paulus HTTP exceptions, it creating the "standard" ones)... but I'm not sure I like that name either, haha. Hmm..